node/builder.go

```diff
@@ -258,6 +258,7 @@ func Online() Option {
 			Override(new(vm.SyscallBuilder), vm.Syscalls),
 			Override(new(*store.ChainStore), modules.ChainStore),
 			Override(new(*stmgr.StateManager), stmgr.NewStateManager),
+			Override(new(stmgr.StateManagerAPI), stmgr.NewStateManager),
```

Comment:
Can do this to avoid creating duplicate instences:
```suggestion
			Override(new(stmgr.StateManagerAPI), From(new(*stmgr.StateManager))),
```